### DISCUSSION TITLE
Live Timing: add pit stop counter

### DISCUSSION THREAD

**Discussion by BeyondVertical** [f1multiviewer/issue-tracker#195] [Feature requests] [answered]
I would like to see the number of pit stops in the timing page. Would be great to have it there.

**Comment by jeppeolesen** [comment, marked as answer]
Pit stops are now included in MultiViewer as a column that can be added to custom live timing layouts. Additionally, if you click the current tyre compound, you can see the stops, how long they took and how many positions a driver lost while stopping.

![Screenshot 2025-03-23 at 00 40 59](https://github.com/user-attachments/assets/ada559e7-8861-4f01-b3d2-034dc0cf6b06)
![Screenshot 2025-03-23 at 00 41 55](https://github.com/user-attachments/assets/e147aa08-0d6a-4a50-9ece-06edc29bb8cc)
